salt/_auth/kubernetes_rbac.py

```diff
@@ -0,0 +1,69 @@
+import base64
+import logging
+
+log = logging.getLogger(__name__)
+
+try:
+    import kubernetes.client
+    import kubernetes.config
```

Comment:
Those import it's only for the futur right ?

Reply:
Ugh, right, I had some code in here before which *did* interact with K8s. I'll remove this.